src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1527,6 +1527,15 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 hostConfig.HostId = (string)hostId;
             }
 
+            // Default AllowHostPartialStartup to true, but allow it
+            // to be overridden by config
+            hostConfig.AllowPartialHostStartup = true;
+            JToken allowPartialHostStartup = (JToken)config["allowPartialHostStartup"];
```

Comment:
Users can now turn this feature off completely if they wish.

Reply:
Can we have a 'full' test where we turn this off with a 'bad' binding? What happens? Does the host throw an exception and the ScriptHostManager goes into it's backoff/retry loop?

Reply:
With this off, you'll get normal SDK behavior. The host will fail to start and will enter the ScriptHostManager host start loop, as it would for any other unhandled host level error.